<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>speedTreeFbxImporter</class>
 <widget class="QWidget" name="speedTreeFbxImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SpeedTree Fbx Importer</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Directory Containing Trees</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="directoryPath">
        <property name="text">
         <string>path/to/dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="launchBrowser">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Trees in Directory</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableOfFoldersOnDisk">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;table of trees tooltip&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <column>
         <property name="text">
          <string>Tree Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fbxs Found</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="onlyImportSelectedFolders">
        <property name="text">
         <string>Only Import Selected Folders</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Tree Subnets in Scene</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listOfTreeSubnets">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <item>
         <property name="text">
          <string>BananaPlant</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BostonFern</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reimportExistingTreeSubnets">
        <property name="text">
         <string>Reimport Existing Tree Subnets</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="convertToYup">
        <property name="text">
         <string>Convert to Y-up Coordinate System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="resetTransformations">
        <property name="text">
         <string>Reset Transformations</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="matchSize">
        <property name="text">
         <string>Match Size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="importFbxExecute">
     <property name="text">
      <string>Import Fbx</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
